src/dispatch/document/flows.py

```diff
@@ -142,8 +148,10 @@ def update_document(document: Document, project_id: int, db_session: SessionLoca
             "document_weblink": resolve_attr(document.incident, "incident_document.weblink"),
             "name": document.incident.name,
             "priority": document.incident.incident_priority.name,
+            "reported_at": document.incident.reported_at.strftime("%m/%d/%Y %H:%M:%S"),
             "resolution": document.incident.resolution,
             "severity": document.incident.incident_severity.name,
+            "stable_at": document.incident.stable_at.strftime("%m/%d/%Y %H:%M:%S"),
```

Comment:
Heads up @Cinojose that you'll need to update the variable names to `reported_at` and `stable_at` on the confluence side when these changes are merged.